src/components/App.js

```diff
@@ -3,6 +3,7 @@ import {
   Route,
   Routes,
 } from 'react-router-dom';
+import './App.css';
```

Comment:
- [ ] The header (navbar) is missing the `avatar` icon, kindly add it.
- [ ] Please make sure that the styling of the header matches the design.

Zeplin Design
![image](https://user-images.githubusercontent.com/33767951/142711840-4e86d45e-8d54-4847-bfbf-7a3a603f4c4e.png)

Your Styling
![image](https://user-images.githubusercontent.com/33767951/142711813-53f39c1c-4843-4cc9-95cb-3914324989c4.png)